src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/ProfileManager.java

```diff
@@ -0,0 +1,156 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.LinearAVF;
+import java.util.HashMap;
+
+public class ProfileManager {
+
+  boolean initialized;
```

Comment:
Je ne vois pas l'intérêt de ce boolean, sauf erreur de ma part, il ne semble être utilisé nulle part sauf dans le Constructeur de ProfileManager

Reply:
Fixed in commit be14aaf